packages/musap-rn-kms/src/agent/MusapKeyManagerSystem.ts

```diff
@@ -120,14 +121,16 @@ export class MusapKeyManagementSystem extends AbstractKeyManagementSystem {
     throw new Error('Not implemented.');
   }
 
-  private asMusapKeyInfo(args: MusapKey): ManagedKeyInfo & { keyUri?: string } {
-    return {
+  private asMusapKeyInfo(args: MusapKey): ManagedKeyInfo {
+    const keyInfo: Partial<ManagedKeyInfo> = {
       kid: args.keyId,
-      kms: args.sscdId,
       type: this.mapAlgorithmTypeToKeyType(args.keyType),
       publicKeyHex: args.publicKey.toString(),
-      keyUri: args.keyUri
-    };
+      meta: {
+        ...args
+      }
+    }
+    return keyInfo as ManagedKeyInfo;
```

Comment:
So we create a partial first and then cast it back to the non partial?

Reply:
yeah, that's the way that we're handling it in the `asSphereonManagedKey` as well. this way we can skip setting the value for `kms` which is mandatory in the veramo `IKey` entity